torch/tensor.py

```diff
@@ -324,6 +324,10 @@ def lu(self, pivot=True, get_infos=False):
         else:
             return LU, pivots
 
+    def nonzero(self, as_tuple=False):
```

Comment:
doesn't this mean we accept as_tuple as a non-kwarg?

Reply:
fixed